corehq/util/datadog/gauges.py

```diff
@@ -29,38 +29,3 @@ def _enforce_prefix(name, prefix):
         "Did you mean to call your gauge 'commcare.{}'? "
         "If you're sure you want to forgo the prefix, you can "
         "pass enforce_prefix=None".format(name))
-
-
-class datadog_track_errors(ContextDecorator):
-    """Record when something succeeds or errors in datadog
-
-    Eg: This code will log to commcare.myfunction.succeeded when it completes
-    successfully, and to commcare.myfunction.failed when an exception is
-    raised.
-
-        @datadog_track_errors('myfunction')
-        def myfunction():
-            pass
-    """
-
-    def __init__(self, name, duration_buckets=None):
```

Comment:
I dropped support for recording the timing info since that was only used in one place in code and never referenced in DD.